From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 10 Nov 2016 20:48:17 +0100
Subject: rename_qcachegrind_icon

---
 qcachegrind/qcachegrind.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qcachegrind/qcachegrind.desktop b/qcachegrind/qcachegrind.desktop
index 81b2102..b12580d 100755
--- a/qcachegrind/qcachegrind.desktop
+++ b/qcachegrind/qcachegrind.desktop
@@ -3,7 +3,7 @@ Type=Application
 Encoding=UTF-8
 Exec=qcachegrind %F
 MimeType=application/x-kcachegrind;
-Icon=kcachegrind
+Icon=qcachegrind
 Terminal=false
 Name=QCachegrind
 Name[bs]=QCachegrind
